src/main/java/org/example/model/ProductVersionRepository.java

```diff
@@ -0,0 +1,71 @@
+package org.example.model;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+import java.util.List;
+
+public class ProductVersionRepository {
+
+    SessionFactory sessionFactory;
+
+    public ProductVersionRepository(SessionFactory sessionFactory) {
+        this.sessionFactory = sessionFactory;
+    }
+
+    public ProductVersion savePVR(ProductVersion productVersion) {
```

Comment:
don't use abbreviation as much as possible, you can use them for EntityManager em or for Exception ex, or something that is commonly acceptable and recognizable but for your internal objects it's better to avoid them. You can imagine how difficult will be onboarding to some project with tens of abbreviations which are confusing.

Reply:
alsso did renaming of variables